### PR TITLE
Sonarr

### LINKED ISSUE #10441: Sonarr
### Brand Name

Sonarr

### Website

https://github.com/Sonarr/Sonarr

### Popularity Metric

Github [link](https://github.com/Sonarr/Sonarr)
> 9.6 K ⭐️

### Official Resources for Icon and Color

https://github.com/Sonarr/Sonarr/blob/develop/Logo/Sonarr.svg
image

I believe the main color is its signature blue #2596be

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by 21st-centuryman**
**Issue:** closes #10441

Github [link](https://github.com/Sonarr/Sonarr)
> 9.6 K ⭐️

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by 21st-centuryman** [comment]
sonarr(1)

**Comment by 21st-centuryman** [comment]
not sure why the preview doesnt work, here is a screenshot
Screenshot 2024-02-11 at 12 48 25

File: icons/sonarr.svg
Comment (adamrusted) [inline icons/sonarr.svg:1]: I'm seeing some issues with your PR when pulled into Affinity Designer, see PR version overlayed in black. There are also some excess points inside the icon that can be removed, and given the shape you've cut out is darker than the outside in the original, I feel we should keep it - but with a 0.5pt cut-out - as shown in the red version below.

image

File: _data/simple-icons.json [hunk 1]
```diff
@@ -13744,6 +13744,11 @@
             "source": "https://www.sonarqube.org/logos/",
             "guidelines": "https://www.sonarqube.org/logos/"
         },
+        {
+            "title": "sonarr",
+            "hex": "2596BE",
+            "source": "https://sonarr.tv"
```
Comment (adamrusted) [inline _data/simple-icons.json:13750]:
```suggestion
            "source": "https://github.com/Sonarr/Sonarr/blob/913b845faadc3c9fc005abfba815426743d01bdf/Logo/Sonarr.svg"
```

As with your other PR, I couldn't find the icon at the source - but I _could_ find it in their git repository.

**Review by adamrusted — CHANGES_REQUESTED** [review]

**Comment by 21st-centuryman** [comment]
sonarr(3)
ok sure, just commited. here is another preview then

File: icons/sonarr.svg
Comment (21st-centuryman) [reply icons/sonarr.svg:1]: check [df3fc44](https://github.com/simple-icons/simple-icons/pull/10444/commits/df3fc447c57a4232acf63169fd7769b958ae29ac)

**Review by adamrusted — APPROVED** [review]

**Pull request merged**